tracer/src/Datadog.Monitoring.Distribution/home/linux-arm64/readme.txt

```diff
@@ -1 +0,0 @@
-﻿Here must go the native binaries for linux-arm64
```

Comment:
I don't know if you you should delete these files - they ensure the directory structure exists correctly on a fresh clone. But if CI works then it's probably fine 🤷

Reply:
I didn't even noticed I had removed those files 😬 . I moved the whole folder but the .gitignore prevented the addition of those. That said, I looked into it and indeed they don't seem needed as the build copies the folder structure for us in https://github.com/DataDog/dd-trace-dotnet/blob/8070df640f6b758fc60f3f7f1942b5afbe2e3694/tracer/build/_build/Build.Steps.cs#L497.
Doing this, I realised that there were some build variables that I can rename as well, I'll do that in an upcoming commit.